### PR TITLE
The shivering symptom now actually chills you like it says it does instead of heating you up

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

Per Cobby's demands, the shivering symptom has been renamed to be the "chills" symptom.

Fixes a copy+paste error by changing the formula of the cap on safe chills's body temperature change to use the body temperature threshold for taking cold damage instead of the body temperature threshold for taking heat damage. The formula will also now never return a positive temperature change for safe chills or a negative temperature change for safe fever.

The caps for the temperature changes of unsafe chills and unsafe fever have been removed.

Safe chills and safe fever should now use an infectee's species's body temperature's actual cold and heat damage thresholds, instead of just using the default ones.

## Why It's Good For The Game

The symptom named ~~"shivering"~~ "chills" should actually make you feel cold.

The cap removals should allow maxed out chills and maxed out fever to make you slightly colder (for humans, maxed out, stage 5 chills will now try to bring you to BODYTEMP_NORMAL - 75 instead of BODYTEMP_NORMAL - 55). The caps didn't appear to server much of a point other than making the chills and fever symptoms even more jank.

## Changelog
:cl: ATHATH
spellcheck: The shivering symptom has been renamed to be the "chills" symptom, per Cobby's request.
fix: The shivering/chills symptom now actually chills you like it says it does instead of heating you up.
fix Safe chills and safe fever should now use an infectee's species's body temperature's actual cold and heat damage thresholds, instead of just using the default ones.
balance: Maxed out, unsafe chills and maxed out, unsafe fevers can now make you slightly colder or hotter than they could before, respectively.
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
but i thought the act of shivering was to produce heat

**Comment by RaveRadbury** [comment]
> but i thought the act of shivering was to produce heat

sounds like the symptom is meant to be called "chills" but was instead labelled "shivering"

**Comment by moocowswag** [comment]
If we wanted to go to realism it would probably be a sweating symptom.

The only other way i could see a virus making someone cool down would be managing to make your metabolic / movement functions so efficient in terms of energy lost to heat generation that you can`t generate much heat on your own.

**Comment by RaveRadbury** [comment]
We have two symptoms that are meant to change your temperature

Sometimes, it's a [fever](https://www.youtube.com/watch?v=aiuFBFPC7YQ) and your temperature goes up.

Sometimes it's a [chill](https://www.youtube.com/watch?v=TXOZRsUXVuw) and your temperature goes down.

The doctor mario school of virology explains it all quite clearly.

**Comment by ATH1909** [comment]
someone else can change the name in the future PR if they want to. that would be out of scope for this PR, though.

**Comment by ExcessiveUseOfCobblestone** [comment]
> The symptom named "shivering" should actually make you feel cold.

no? shivering is CAUSED by being cold but its actually a response to warm you up. being cold CAUSES shivering.

The fix is to "correct" the functionality on top of changing the name (because it now has different functionality), both seem to be in scope here.

**Comment by ATH1909** [comment]
> The fix is to "correct" the functionality on top of changing the name (because it now has different functionality), both seem to be in scope here.

a name change would mean going through the wiki and through all of my old viro recipes to replace all mentions of "shivering" with "chills" or whatever the new name would be.

again, you can PR that name change yourself, but I'm not going to change a name that is several years old just because I fixed a fairly recently-created bug. I'm not just saying this for "muh soul" reasons- the wiki and all of my old viro recipes would have to be updated to incorporate the new name, and that's _effort_.

**Comment by LemonInTheDark** [comment]
Isn't the point of shivering to heat your body up?

**Comment by ATH1909** [comment]
> Isn't the point of shivering to heat your body up?

aaaaaaaAAAAAAAAAAHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

**Comment by ExcessiveUseOfCobblestone** [comment]
Shivering may have cooled people down for years in the game but I think shivering as a real bodily response has lasted longer.

**Comment by ATH1909** [comment]
> Shivering may have cooled people down for years in the game but I think shivering as a real bodily response has lasted longer.

then you can change that in YOUR PR. I'm not adding a name change to a bug fix PR.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1575,6 +1575,14 @@
 		return BODYTEMP_NORMAL
 	return BODYTEMP_NORMAL + get_body_temp_normal_change()
 
+///Returns the body temperature at which this mob will start taking heat damage. Gets overridden in carbon/human/life.dm. Not actually intended for usage by non-carbons; this proc being defined at the living level is just to match get_body_temp_normal() (and for safety's sake).
+/mob/living/proc/get_body_temp_heat_damage_limit()
```
Comment (Fikou) [inline code/modules/mob/living/living.dm:1579]: isnt what this stuff will return fake for simplemobs or like aliens?

**Comment by Fikou** [comment]
also the point of shivering is to heat your body up

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (ATH1909) [reply code/modules/mob/living/living.dm:1579]: the comments of this proc explicitly mention that it should only be used with carbons

I'll move it down to the carbon level, at least

Comment (ATH1909) [reply code/modules/mob/living/living.dm:1579]: actually, hm. get_body_temp_normal() is up here at the living level, so maybe this should remain here as well

**Comment by ExcessiveUseOfCobblestone** [comment]
It's not bike shed when you are creating a disparity between how something functions in game vs how something functions in real life. I can't stress this enough but it is literally the opposite of what you'd expect, as again shivering means you are already cold and the body tries to warm you up. Shivering does not make you code.

I consider the Name-function inconsistency and the lack of a cooling symptom different aspects of the same issue. Fixing the latter in a way that creates an issue with the former and having that pointed out is NOT bikeshed and inscope if someone is trying to tweak it. how it worked in the past does not make it not an issue/oversight/inconsistency error.

If you don't want to do a proper fix to the issue fine but I will be holding off on merging until then.

**Comment by ATH1909** [comment]
> It's not bike shed when you are creating a disparity between how something functions in game vs how something functions in real life. I can't stress this enough but it is literally the opposite of what you'd expect, as again shivering means you are already cold and the body tries to warm you up. Shivering does not make you cold.
> 
> I consider the Name-function inconsistency and the lack of a cooling symptom different aspects of the same issue. Fixing the latter in a way that creates an issue with the former and having that pointed out is NOT bikeshed and inscope if someone is trying to tweak it. how it worked in the past does not make it not an issue/oversight/inconsistency error.
> 
> If you don't want to do a proper fix to the issue fine but I will be holding off on merging until then.

but I'm not creating this disparity, this disparity was _already there_.

**Comment by ExcessiveUseOfCobblestone** [comment]
Im not saying you did, im saying im not going to let you retain the disparity going forward.

**Comment by ATH1909** [comment]
FINE, cobby, you've successfully strongarmed me into changing the name of the "shivering" symptom to "chills". can you stop strangling this PR now?

**Comment by ATH1909** [comment]
oh god fucking damnit why the hell isn't it compiling

**Comment by ATH1909** [comment]
there, fixed it

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -1575,6 +1575,14 @@
 		return BODYTEMP_NORMAL
 	return BODYTEMP_NORMAL + get_body_temp_normal_change()
 
+///Returns the body temperature at which this mob will start taking heat damage. Gets overridden in carbon/human/life.dm. Not actually intended for usage by non-carbons; this proc being defined at the living level is just to match get_body_temp_normal() (and for safety's sake).
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/living.dm:1578]:
```suggestion
///Returns the body temperature at which this mob will start taking heat damage.
```

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -1575,6 +1575,14 @@
...
+/mob/living/proc/get_body_temp_heat_damage_limit()
+	return BODYTEMP_HEAT_DAMAGE_LIMIT
+
+///Returns the body temperature at which this mob will start taking cold damage. Gets overridden in carbon/human/life.dm. Not actually intended for usage by non-carbons; this proc being defined at the living level is just to match get_body_temp_normal() (and for safety's sake).
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/living.dm:1582]:
```suggestion
///Returns the body temperature at which this mob will start taking cold damage.
```

File: code/modules/mob/living/living.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/living.dm:1578]: if the things i deleted are true then it has no right being here.

**Pull request merged**